src/const/restaurantList.js

```diff
@@ -0,0 +1,48 @@
+const restaurants = [
+    {
+      id: 1,
+      name: "피양콩할마니",
+      description:
+        "평양 출신의 할머니가 수십 년간 운영해온 비지 전문점 피양콩 할마니. 두부를 빼지 않은 되비지를 맛볼 수 있는 곳으로, ‘피양’은 평안도 사투리로 ‘평양’을 의미한다. 딸과 함께 운영하는 이곳에선 맷돌로 직접 간 콩만을 사용하며, 일체의 조미료를 넣지 않은 건강식을 선보인다. 콩비지와 피양 만두가 이곳의 대표 메뉴지만, 할머니가 옛날 방식을 고수하며 만들어내는 비지전골 또한 이 집의 역사를 느낄 수 있는 특별한 메뉴다. 반찬은 손님들이 먹고 싶은 만큼 덜어 먹을 수 있게 준비돼 있다.",
+      category: "한식",
+      img_src: "../../../templates/category-korean.png",
```

Comment:
```suggestion
      imgSrc: "../../../templates/category-korean.png",
```

JavaScript에서는 변수명, 객체의 속성명에는 주로 카멜 케이스를 사용하고, 대부분의 컨벤션에서도 카멜 케이스를 사용하므로 특별한 상황이 아닌 경우 카멜 케이스의 사용을 제안드립니다. 어떠한가요?

물론, 몇몇 사유로 카멜 케이스를 어쩔 수 없이 사용하는 경우도 있어요. 예를 들면 외부 API에서 반환하는 데이터가 간혹가다가 카멜 케이스가 아닌 경우가 있는데, 이 경우에는 상황에 따라 반환받은 값을 관리하기 위해 예외적으로 카멜 케이스를 사용하지 않기도 합니다!

Reply:
아 그렇네요! 함수명 등을 카멜케이스로 적는 것은 익숙하지만, 어째서인지 변수명 등에서 두 단어가 올 경우 자꾸만 '_'를 이용하게 되네요.. 카멜케이스의 사용에 좀 더 주의를 기울여봐야겠습니다.